misc/parity/test_indexer_v_algod.py

```diff
@@ -100,7 +88,7 @@ def ddize(d):
             return [ddize(x) for x in d]
         return d
 
-    return ddize(prettify_diff(diff, src="ALGOD", tgt="INDEXER", value_limit=30))
+    return ddize(prettify_diff(diff, src="ALGOD", tgt="INDEXER", value_limit=50))
```

Comment:
Is it possible to have an unbounded diff? It could be useful for eyeballing all the diffs locally, but I'm not sure if it's too verbose.